frontend/src/components/submission/submission-form.tsx

```diff
@@ -49,11 +49,12 @@ const PaperForm = () => {
   const onSubmit = async (data: z.infer<typeof schema>) => {
     try {
       const response = await fetch(
-        `${process.env.NEXT_PUBLIC_API_URL}/article/new`,
+        `${process.env.NEXT_PUBLIC_API_URL}/articles/new`,
         {
           method: 'POST',
           headers: {
             'Content-Type': 'application/json',
+            'Authorization': "Bearer " + localStorage.getItem("token")
```

Comment:
./src/contexts/auth-context.tsx useAuth() exports token and can be used here

Reply:
added useAuth, changed to `Bearer ${token}`